### PR TITLE
perf: reduce bundle size

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

Fixes # (issue)

- removed components, packages and pages that we are not using

**Environment**: Staging(main branch) / Production

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

### CODE REVIEW

File: apps/web/pages/_app.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Script from "next/script";
 
 import "@calcom/embed-core/src/embed-iframe";
 import { useEmbedUiConfig } from "@calcom/embed-core/src/embed-iframe";
-import LicenseRequired from "@calcom/features/ee/common/components/LicenseRequired";
+import LicenseRequired from "@calcom/features/ee/common/components/v2/LicenseRequired";
```
Comment (zomars) [inline apps/web/pages/_app.tsx:9]: Let's replace the v1 with the v2 instead 🙏🏽

File: packages/features/ee/common/components/v2/LicenseRequired.tsx
Comment (zomars) [inline packages/features/ee/common/components/v2/LicenseRequired.tsx:1]: Moved to v1 to replace it for good.

**Comment by baileypumfleet** [comment]
I think there is also a /sandbox page that should be able to be removed

File: packages/features/auth/PermissionContainer.tsx [hunk 2]
```diff
@@ -19,7 +19,8 @@ export const PermissionContainer: FC<AdminRequiredProps> = ({
   const session = useSession();
 
   // Admin can do everything
-  if (session.data?.user.role !== roleRequired || !UserPermissionRole.ADMIN) return null;
+  if (session.data?.user.role !== roleRequired && session.data?.user.role != UserPermissionRole.ADMIN)
```
Comment (hariombalhara) [inline packages/features/auth/PermissionContainer.tsx:22]: @sean-brydon Fixed this check.
cc @Udit-takkar. Please test this too.

Comment (Udit-takkar) [reply packages/features/auth/PermissionContainer.tsx:22]: nice catch

File: packages/features/ee/teams/components/TeamAvailabilityModal.tsx [hunk 3]
```diff
@@ -29,66 +33,71 @@ export default function TeamAvailabilityModal(props: Props) {
...
-              size="sm"
-              imageSrc={WEBAPP_URL + "/" + props.member?.username + "/avatar.png"}
-              alt={props.member?.name || ""}
-              className="h-14 w-14 rounded-full"
-            />
-            <div className="ms-3 inline-block pt-1">
-              <span className="text-default text-lg font-bold">{props.member?.name}</span>
-              <span className="text-muted -mt-1 block text-sm">{props.member?.email}</span>
+      <>
+        <div className="grid h-[400px] grid-cols-2 space-x-11 rtl:space-x-reverse">
```
Comment (hariombalhara) [inline packages/features/ee/teams/components/TeamAvailabilityModal.tsx:37]: What are these UI Changes? Is there a ticket for it?

Comment (hariombalhara) [reply packages/features/ee/teams/components/TeamAvailabilityModal.tsx:37]: got my answer. It's a file movement from v2.

**Review by hariombalhara — APPROVED** [review]
Screenshot 2023-04-18 at 5 40 21 PM
Don't see much improvement in bundle size but nevertheless a very welcome change.

**Review by sean-brydon — APPROVED** [review]
Thought i reviewed this already :P 

great job

@hariombalhara good spot on the PermissionContainer 🙏

**Pull request merged**